thirdparty/cmdconfig/commands/cmdeval/cmdeval.go

```diff
@@ -57,8 +57,6 @@ func GetEvalFnRunner(ctx context.Context, parent string) *EvalFnRunner {
 		&r.Exec, "exec", "", "run an executable as a function")
 	r.Command.Flags().StringVar(
 		&r.FnConfigPath, "fn-config", "", "path to the function config file")
-	r.Command.Flags().BoolVarP(
-		&r.IncludeMetaResources, "include-meta-resources", "m", false, "include package meta resources in function input")
```

Comment:
If we want to print a helpful message when they are using this flag, we can't remove these lines. As you asked, I created a draft PR for how to gracefully let users know this is no longer available.

See https://github.com/GoogleContainerTools/kpt/pull/2939

Reply:
Yes, re-introduced `include-meta-resources` as hidden flag and throwing an error as suggested in your PR. Thanks.